icons/fonts/index.html

```diff
@@ -13386,9 +13386,9 @@
     </div>
     <div class="icon-card">
       <div class="icon-container">
-        <div class="icon icon-queue-contact_20"></div>
+        <div class="icon icon-Queue-contact_20"></div>
```

Comment:
I don't know why this is now upper case Q again. This causes issues. We need to keep all icons lowercase

Reply:
I found why? In the SVG folder, the SVG file's name is not lowercase. So every time you generate icons, it will use that name. We need rename the icon under SVG folder or update the generating tool.
![image](https://user-images.githubusercontent.com/4829803/114964174-c612aa00-9ea0-11eb-9ab0-f581828875f3.png)